Update voice-tone-and-writing-style-guidelines.md (#78)
mattermost/mattermost-handbook: operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md

# Voice, Tone and Writing Style Guidelines

Below is a start of voice, tone and writing style guidelines for public web properties. This is a work in progress, to be iterated on. Some of the content in this document is borrowed from the [Mattermost Editorial Style Guide](https://docs.google.com/document/d/1XWjtWdF77qKdxDso_-aC_S1c3E0ohOoxCRL_PIf3pco/edit#), which is managed by the Marketing team. 

Some of the recommendations across the documents, such as punctuating lists and heading capitalization are not currently aligned but will be iterated on for alignment across all web properties. 

## Voice and Tone

Our voice is informed by our foundation as a messaging platform for Enterprise Developers and DevOps, but respects that our client base spans many types of organizations. Writing is one way we can bring Mattermost’s values to life. And above all, our voice demonstrates our deep respect for our clients and users. Ultimately, Mattermost exists because of them, so it’s important we keep their needs at the forefront of our writing style.

> Mattermost's voice is:

Reviewer: Would you order these by priority, or alphabetically?

Author: I think they're all equally so alphabetically would probably be most fair.

Author: Updated to alphabetical.

* Professional
* Concise
* Clear
* Consistent
* Customer-centric

> * Empowering

Reviewer: Would you have an example of what an "empowering" voice means?

Author: I believe "Empowering" in this sense would mean to provide the reader with a sense of being empowered to use Mattermost so maybe "Empowering" should be changed to something like "Guiding" or "Confident"?

To achieve this, we keep the following principles in mind when writing content: 

* Use simple, plain language that is easy to understand
* Use direct, casual tone instead of an informal tone
* Use the active voice

> * Write clearly - avoid vague terms and jargon

Reviewer: Wondering if the two new principles have overlap with the existing ones? E.g. 'Write clearly - avoid vague terms and jargon' overlaps with 'Use simple, plain language that is easy to understand'

Author: Good point. 

Point one can be: "Use simple, plain language that is easy to understand and avoid jargon" and then the new bullet point can be removed.

* Write short, active sentences and maintain a visual separation between page elements

The appropriate tone differs across different mediums. We don’t write help documentation with the same tone as website copy.

> You can vary your tone to fit the situation, just as you’d talk to an angry person with a different tone than with an excited child. Voice is to tone as climate is to weather. The Mattermost voice remains the same, even when the tone varies.

Reviewer: `The Mattermost voice remains the same, even when the tone varies.` -- excellent point. 0/5, may be worth having it as a separate paragraph for it to stand out.

Author: I can't take credit for that - it's from the marketing editorial guide :) and a new paragraph would definitely work.

> ## Writing Tenets

Reviewer: Do you mean writing principles?

> ## Writing Tenets

Author: ```suggestion
## Writing Principles
```

> * Work on the premise that "Every page is page 1", as a large portion of users enters our documentation from a Google search**

Reviewer: ```suggestion
* Work on the premise that "Every page is page 1", as a large portion of users enters our documentation from a Google search
```

> * Work on the premise that "Every page is page 1", as a large portion of users enters our documentation from a Google search**

Reviewer: Same as in the principles above, wondering if there is overlap with any listed here, e.g. 'Break long sections into smaller, easily digestable subpages', and 'Minimize content so it can be found and remembered. Keep pages short, modular, and focused on a single topic'

* Add a summary to the top of each page for readers to be able to quickly assess the content for suitability
* Write clearly and concisely - get to the point quickly without losing valuable information
* Focus on what the target audience wants to accomplish by being practical/outcome-focused
* Break long sections into smaller, easily digestable subpages
* Have at most one key point or action per paragraph

> * Refer to one thing or idea with the same word throughout the page.

Reviewer: Some bullets end in a period like here, while others do not

* Use ordered lists or bullets where appropriate, as they are generally easier to read than long blocks of text
* Write for an international audience without idioms or expressions that people outside of your region are unlikely to understand
* Minimize content so it can be found and remembered. Keep pages short, modular, and focused on a single topic.

> ## Writing Style

Reviewer: Out of curiosity, what are your thoughts on how the existing docs style guidelines will fit with these updated guidelines?

Will they over time migrate to this one, or would we keep them separate (this being a general voice/writing guidelines across web properties, with specific guidelines for docs separately)

https://docs.mattermost.com/guides/core.html#documentation-style-guide

Author: Good question. This section was an attempt to distill the "ideal basics" such as numbering and I considered adding more details about markdown/rst formatting but imo that is too detailed for this guide and can be a separate "House Style" guide for use by editors and technical writers. The nitty-gritty of writing is barrier to contribution so the current doc guidelines will be a separate section in the publishing space. 

Taking this into account, it may be better to cut out this section and refer to the "house style guide" instead (when it's available) so there's less cognitive load. 

What are your thoughts?

Reviewer: Hi @justinegeffen - an excellent point. I like the idea of a style guide for editors.

For this section, if there are rules or guidelines that would apply across our web properties, e.g. numbers and capitalization may be, then those are okay to keep. But more detailed guidelines around markdown/RST fit better for the docs-specific guidelines for editors.

Author: The two below points apply across all web properties going forward - and existing content will be updated to align with this in a phased approach (to be discussed). 

Agree that more detailed guidelines are appropriate in an editorial context but not here.

Reviewer: Thanks! Agree

**Numbers**

Spell out numbers when the number is the first word in a sentence or is less than or equal to ten, otherwise use numeric digits. Use commas to make long numbers easier to read.

**Capitalization**

> Use title case for page titles and section titles.

Author: ```suggestion
Use sentence case for page titles, headings, and section titles (e.g. "Writing guidelines for editors").
```

> Title case: Grammar, Spelling, and Mechanics

Author: Remove.

> Sentence case: Language and spelling

Author: Remove.

> ## Avoid Fluff

Reviewer: "fluff" may not be easy to understand for non-English speakers

Author: I agree - but struggled to come up with a better word. I'm looking at moving this section into a more appropriate place and removing this heading.

Author: Resolved.

We avoid using the following words and phrases as they're overused and vague:

* Work better
* Do better work, faster
* Get more done in less time
* Chat (as a stand alone feature reference. However it is ok to use chat when describing a benefit "Create a Jira ticket without leaving your chat window" )
* DevOps teams 
* Utilize (instead, use “use”)
* High performance teams
* Phrases that directly praise ourselves: “We’ve built an intuitive workplace messaging solution”, “It’s a joy to use”

> ## Writing an MVP

Reviewer: Wondering if this section belongs here or in a separate page?

Author: It would probably work better in its own sub-section (similar to the "House Style Guide"). Agree.

> When you submit a PR for your contribution, please include documentation with it. The documentation can be submitted as a new file using the following format:

Reviewer: Propose writing an example of what a sample "MVP" would look like - or referencing an existing example

Author: Good point. 

Next step for this is to highlight it - maybe using a code block. And then putting together an actual example which would probably also ensure there's no confusion as to the content.

\## Configuration (if applicable)\
*Provide configuration and set up steps in the context of the change/feature.*

\## Code Samples (if applicable)\
*Include any code samples that should be included in the user-facing documentation.*

\## Installation Process\
*Include any processes or procedures that users have to perform to enable/use the change/addition.*

\## Troubleshooting/Known Issues (if applicable)\
*Include any known issues or limitations that may affect how the feature/process works.*

We don't expect a huge body of documentation or that it's perfectly-written - but rather a clear, concise outline of the change which can be added to our documentation. 

During the editorial review, editors may make punctuation and/or terminology changes and commit them to save time on the review process. This only applies to punctuation/terminology - any content suggestions and questions will follow the usual review and discussion process.

Once the editorial review is complete, we'll move your documentation contribution to the appropriate part of the guide (if it's not there already). We'll share the URL and you can edit it at any time if you need to.

## More Information

> These guidelines were inspired by [Stripe's knowledge base content](https://document360.io/blog/tear-down-of-stripe-knowledge-base/), which is widely perceived as excellent by the developer community, and [by other sites shared in this Document360 article](https://document360.io/blog/10-knowledge-base-software-best-practice-examples/).

Reviewer: This is related to my changes, but re-reading it, wondering if we need to say `widely perceived as excellent` and just keep it simple:

```
These guidelines were inspired by [Stripe's knowledge base content](https://document360.io/blog/tear-down-of-stripe-knowledge-base/) and [by other sites shared in this Document360 article](https://document360.io/blog/10-knowledge-base-software-best-practice-examples/).
```

Author: Done.
